src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,649 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures aggregation expression performance.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE aggregation expressions and numbers on the 'standalone' environment
+  for the classic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 64
```

Comment:
[nit] I see below that the highest phase number is 62, so this should be 63 (unless there are duplicate phase ids; please double check that this isn't the case)

Reply:
Done.